dialogue-example/src/main/java/com/palantir/dialogue/example/SampleService.java

```diff
@@ -16,8 +16,43 @@
 
 package com.palantir.dialogue.example;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.palantir.ri.ResourceIdentifier;
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.Objects;
+
 // Example of the interface code conjure would generate for a simple SampleService.
 public interface SampleService {
-    String stringToString(String objectId, String header, String body);
+
+    final class SampleObject {
```

Comment:
Conjure probably doesn't end up nesting this object in generated code?

Reply:
Correct. I'll move it up.